tests/integration/components/light-table-test.js

```diff
@@ -189,3 +189,29 @@ test('passed in components can have computed properties', function(assert) {
 
   assert.equal(this.$('.custom-row.is-active').length, 0, 'none of the items are active');
 });
+
+test('onScroll', function(assert) {
+  let done = assert.async();
+  let table = new Table(Columns, createUsers(10));
+
+  this.setProperties({
+    table,
+    currentScrollY: 0,
+    onScroll() {
+      assert.ok(true, 'onScroll worked');
```

Comment:
Done 😃 .

`300` was indeed too much scrolling for this test, `50` does the trick.